setup.py

```diff
@@ -1,16 +1,17 @@
-from setuptools import setup
-from setuptools import find_packages
+import setuptools
 
-setup(name='hetmech',
-      description='A search engine for hetnets',
-      long_description='Matrix implementations of path-count-based measures',
-      url='https://github.com/greenelab/hetmech',
-      license='BSD 3-Clause License',
-      packages=find_packages(),
-      install_requires=[
-          'hetio>=0.2.8',
-          'numpy',
-          'scipy',
-          'xarray',
```

Comment:
Without xarray, [`hetmatpy.xarray`](https://github.com/hetio/hetmatpy/blob/8ee8b3707fef3bdfe3495ed0c1132645fc95b05c/hetmatpy/xarray.py#L2) won't work.

Reply:
Yes. If you want to use that submodule, you will have to install xarray. I didn't want to force this dependency everywhere, since it's a non-essential aspect of the package.